statics/src/main/java/com/codeborne/selenide/Selenide.java

```diff
@@ -582,10 +582,20 @@ public static ElementsCollection getElements(By criteria) {
     return getSelenideDriver().findAll(criteria);
   }
 
+  /**
+   * @see @see <a href="https://selenium.dev/selenium/docs/api/java/org/openqa/selenium/JavascriptExecutor.html#executeScript-java.lang.String-java.lang.Object...-">executeScript</a>
```

Comment:
```suggestion
   * @see <a href="https://selenium.dev/selenium/docs/api/java/org/openqa/selenium/JavascriptExecutor.html#executeScript-java.lang.String-java.lang.Object...-">executeScript</a>
```

Looks like checkstyle isn't pass such long line. We have a limit equals to `136` characters.